docs/configuration/module/serial.mdx

```diff
@@ -11,7 +11,7 @@ The serial module config options are: Enabled, Echo, Mode, Receive GPIO, Transmi
 
 This is an interface to talk to and control your Meshtastic device over a serial port. The module can be set to different [modes](#mode), each fulfilling a different use-case.
 
-![image](https://user-images.githubusercontent.com/9000580/205529843-962c3187-8411-452c-b729-42c58b1571f5.png)
+![image](https://user-images.githubusercontent.com/9000580/205529843-962c3187-8411-452c-b729-42c58b1571f5.webp)
```

Comment:
This links out of Meshtastic.  A build attempt may find them all.

Reply:
Build may not find missing images but it does find:
  Exhaustive list of all broken links found:
  - Broken link on source page path = /docs/software/integrations/mqtt/nodered:
     -> linking to /documents/mqtt/NodeRedTwo.webp
     -> linking to /documents/mqtt/NodeRedThree.webp
     -> linking to /documents/mqtt/NR_nodes.webp
     -> linking to /documents/mqtt/Broker1.webp
     -> linking to /documents/mqtt/Consume.webp
     -> linking to /documents/mqtt/Inject.webp
     -> linking to /documents/mqtt/Forward.webp
     -> linking to /documents/mqtt/DecodeNewest.webp
     -> linking to /documents/mqtt/Mapping.webp
     -> linking to /documents/mqtt/EncodingPosition.webp
     -> linking to /documents/mqtt/PosJSON.webp

Reply:
🤦 not sure how I missed this. Didn't see the build errors I guess. Thanks

Reply:
Do you know why these images are under `documents` instead of `static/img`?

Reply:
that page is pretty old, i'm sure it'd be safe to move em.